js/tabRestore.js

```diff
@@ -0,0 +1,33 @@
+const MAX_TAB_HISTORY_DEPTH = 20
+var tabHistoryStack = Array()
```

Comment:
You make a good point! The reason I don't do that is because I don't think the namespace pattern, as it is used here, allows it. Quite a bit of the source uses this same pattern, actually (take js/tabState.js for example). There are more careful approaches we could use to avoid naming collisions but I don't think it's an issue here.